README.md

```diff
@@ -18,8 +16,14 @@ gem "slim-rails"
 
 And that's it.
 
-If you have existing `.erb` templates, check out [our guide](https://github.com/slim-template/slim/wiki/Template-Converters-ERB-to-SLIM) on how to achieve this.
+Every time you generate a controller or scaffold, you'll get Slim templates.
 
-From the version 0.2.0, there is no need to include gem "slim" in your Gemfile.
+If you have existing `.erb` templates, check out our guide [How to convert your `.erb` templates to `.slim`](https://github.com/slim-template/slim/wiki/Template-Converters-ERB-to-SLIM) on how to achieve this.
 
-Every time you generate a controller or scaffold, you'll get Slim templates.
+This gem is tested with Ruby on Rails 5.2+.
```

Comment:
If we go this route, maybe also include a link to the CI matrix file for people to review which versions are tested.

Reply:
Linked!